src/ng/rootScope.js

```diff
@@ -437,7 +437,7 @@ function $RootScopeProvider() {
        * If any one expression in the collection changes the `listener` is executed.
        *
        * - The items in the `watchExpressions` array are observed via standard $watch operation and are examined on every
-       *   call to $digest() to see if any items changes.
+       *   call to $digest() to see if any items change.
```

Comment:
What if we put it this way?

*The items in the `watchExpressions` array are observed via standard `$watch` operation. Their return values are examined for changes on every call to `$digest`.*